docs/deployment/getting_started_locally_with_extensions.md

```diff
@@ -0,0 +1,160 @@
+# Deploying Gardener locally and enable provider-extensions
```

Comment:
```suggestion
# Deploying Gardener locally with seed on existing Kubernetes cluster
```

IIUC, this matches the primary intent of this setup.

Reply:
The first title includes the purpose, the second one the setup.
I prefer the first one because it gives you an idea when you should use this scenario.